src/assets/styles/_display.scss

```diff
@@ -0,0 +1,79 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+  -webkit-tap-highlight-color: transparent;
+  min-height: 100%;
+}
+
+body {
+  table {
+    width: 100%;
+  }
+}
+
+.container {
+  max-width: 74rem;
+  width: 90%;
+}
+
+.mw400 {
+  max-width: 400px;
+}
+
+.mw500 {
+  max-width: 500px;
+}
+
+.mw600 {
+  max-width: 600px;
+}
+
+.block {
+  display: block;
+}
+
+.inline {
+  display: inline;
+}
+
+.in-block {
+  display: inline-block;
+  zoom: 1;
+}
+
+.d-none,
+.hide,
+.hidden {
+  display: none;
+}
+
+.t-0 {
+  top: 0;
+}
+
+.b-0 {
+  bottom: 0;
+}
+
+.l-0 {
+  left: 0;
+}
+
+.r-0 {
+  right: 0;
+}
+
+.tb-0 {
+  top: 0;
+  bottom: 0;
+}
+
+.lr-0 {
+  left: 0;
+  right: 0;
+}
+
+.fullwidth {
```

Comment:
We have a separate scss file for width and height which makes this redundant (you can combine them or keep both and remove redundancies)